sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/ViewBuilder.java

```diff
@@ -6,19 +6,39 @@
 package io.opentelemetry.sdk.metrics.view;
 
 import io.opentelemetry.sdk.metrics.aggregator.AggregatorFactory;
+import io.opentelemetry.sdk.metrics.processor.LabelsProcessorFactory;
 import java.util.Objects;
 
 public final class ViewBuilder {
   private AggregatorFactory aggregatorFactory;
+  private LabelsProcessorFactory labelsProcessorFactory = LabelsProcessorFactory.noop();
 
   ViewBuilder() {}
 
+  /**
+   * sets {@link AggregatorFactory}.
+   *
+   * @param aggregatorFactory aggregator factory.
+   * @return this Builder.
+   */
   public ViewBuilder setAggregatorFactory(AggregatorFactory aggregatorFactory) {
     this.aggregatorFactory = Objects.requireNonNull(aggregatorFactory, "aggregatorFactory");
     return this;
   }
 
+  /**
+   * sets {@link LabelsProcessorFactory}.
+   *
+   * @param labelsProcessorFactory labels processor factory.
+   * @return this Builder.
+   */
+  public ViewBuilder setLabelsProcessorFactory(LabelsProcessorFactory labelsProcessorFactory) {
+    this.labelsProcessorFactory =
+        Objects.requireNonNull(labelsProcessorFactory, "labelsProcessorFactory");
```

Comment:
Not necessary to have this and check for aggregator as well, I think the auto-value generated class will do the checks for you

Reply:
agreed. Done

Reply:
Not done.